apache/superset
# fix: Bad date type in email text report for table chart

### SUMMARY


When creating a report that sends a table chart with dates, and choose the option send by email as text, the dates show badly formatted

It is because `/api/v1/chart/data` endpoint returns datetime64 data which represents as milliseconds.

Need to convert is as date type when we create pandas Dataframe from the result data.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

BEFORE:
![image](https://user-images.githubusercontent.com/39701522/169124937-5248dca6-186f-4d35-a048-48831781cf7e.png)

AFTER: 
<img width="318" alt="image" src="https://user-images.githubusercontent.com/39701522/169124244-8da4661d-03a7-4114-aca8-5a657b2e190e.png">


### TESTING INSTRUCTIONS

1. Create a table chart which contains date.
2. Create a text email report.
3. Check your email.

### ADDITIONAL INFORMATION
